docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+### Create an IAP configuration document
+
+This document represents a list of Play Store plans (identified uniquely by their Play Store [`token`](https://developer.android.com/google/play/billing#tokens) for your product.
+
+1. Go into Google Play Console and find details about your android app by [`token and orderId`](https://developer.android.com/google/play/billing#tokens).
+2. Create a configuration document similar in form to https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/guardian-vpn (omitting iOS plans) using the details retrieved in Step 1. This can be in a Google document for now.
+   - Note: The only information SubPlat uses from the VPN document currently is the `productId` and `platform`. The VPN configuration document is structured in this way for historical reasons. We may change how this information is structured for future integrations.
+3. File a ticket in the [FXA Jira project](https://mozilla-hub.atlassian.net/browse/FXA) to create a new document in SubPlat's "IAP config" Firestore collection, and provide the document from Step 2 along with an `appName`.
+   - `appName` is a human-readable name that SubPlat uses to uniquely identify your app in our system (e.g. `"guardian-vpn"` is the `appName` for the Mozilla VPN in the link above).
+     When the ticket is complete and deployed, you should be able to view your configuration at https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/${appName}.
+
+### Add Google Play Developer API keys in SubPlat
+
+A SubPlat engineer with developer access to Google Play Console (see [Prerequisites](#prerequisites)) can configure this for you when ready.
+
+Google Play Developer API keys are needed for SubPlat to make calls to Google API. See [Google IAP Integration](/ecosystem-platform/tutorials/subscription-platform#google-iap-integration) for more information on how to obtain these keys.
+
+### Send Real-time developer notifications to SubPlat
+
+A SubPlat engineer with developer access to Google Play Console (see [Prerequisites](#prerequisites)) can configure this for you when ready.
+
+The Sandbox and, when ready, Production endpoints in Real-time developer notifications must be updated with SubPlat's notification endpoint in order for our system to receive Real-time developer notifications.
+
+### Map Google IAP to Stripe plans
+
+In order to know what capabilities to grant a given Google IAP user, we map Play Store `productId`s to [Stripe `price` or `plan`](https://support.stripe.com/questions/how-to-create-products-and-prices) IDs (`price`s supersede `plan`s).
+
+1. Go into the Google Play Console and find your android app’s human-readable `productId`s.
```

Comment:
Ah I guess Google renamed `sku` to `productId`? I think it's worth mentioning everywhere you reference `productId` that it was formerly known as `sku` (since that's how we reference it in our code base)

Reply:
Confirmed. Updated as mentioned above.